packages/console/src/pages/WebhookDetails/index.tsx

```diff
@@ -52,7 +52,7 @@ function WebhookDetails() {
   const [isDeleting, setIsDeleting] = useState(false);
   const [isDisableFormOpen, setIsDisableFormOpen] = useState(false);
   const [isUpdatingEnableState, setIsUpdatingEnableState] = useState(false);
-  const [isEnabled, setIsEnabled] = useState(data?.enabled ?? true);
+  const isEnabled = data?.enabled;
```

Comment:
nit: Either remove this line and use `data.enabled` in line 87, or destructure both `enabled` and `executionStats` in here

Reply:
Rebased.